crates/pink/examples/hooks_test/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+name = "hooks_test"
+authors = ["Kevin Wang <wy721@qq.com>"]
+version = "0.1.0"
+edition = "2018"
+resolver = "2"
+
+[dependencies]
+ink_primitives = { version = "3.0.0-rc3", default-features = false }
```

Comment:
Look like the latest version is `rc6`. Will we switch?

Reply:
I think it doesn't metter. The version is generated by cargo-contract and it is actual resolved to rc6 at build time.

Reply:
Ack